projects/packages/my-jetpack/_inc/components/testimonials/style.scss

```diff
@@ -0,0 +1,75 @@
+
+.testimonial-base {
+    flex: 1;
+    background-size: cover;
+    background-position: center top;
+    background-repeat: no-repeat;
+    font-family: 'Open Sans', sans-serif;
```

Comment:
Please use SCSS variables from Jetpack that already have font names and colours defined.

Are all those CSS rules needed to achieve the desired effect?